### PR TITLE
addPreviousAndNextButton

### PR DESCRIPTION
**Pull request by rebecalsantos**
Added Previous and Next week Button in  WeeklyTable

### CODE REVIEW

File: client/src/pages/WeeklyTable.js [hunk 1]
```diff
@@ -74,12 +76,55 @@ const WeeklyTable = ({
 	refreshBooking,
 	maxDesksForDay,
 }) => {
-	const bookingsByRow = getBookingsByRow(bookings, rowsCount, week);
-	console.log({ bookingsByRow });
+	const [currentMonday, setCurrentMonday] = useState(new Date(2022, 0, 17));
+	let week = [];
+	const year = currentMonday.getFullYear();
+	const month = currentMonday.getMonth();
+	let date = currentMonday.getDate();
```
Comment (Benjamin-GitHub) [inline client/src/pages/WeeklyTable.js:83]: I'm wondered why shouldn't use "let" either for month and year!

File: client/src/pages/WeeklyTable.js [hunk 2]
```diff
@@ -74,12 +76,55 @@ const WeeklyTable = ({
...
 	const bookingsByDesk = getBookingsByDesk(bookings, week);
+	// function setThisMonday() {
+
+	// }
+	function setNextMonday() {
+		setCurrentMonday((currentMonday) => {
+			return new Date(
+				currentMonday.getFullYear(),
+				currentMonday.getMonth(),
+				currentMonday.getDate() + 7
```
Comment (Benjamin-GitHub) [inline client/src/pages/WeeklyTable.js:101]: I can see here the reoson of my question ;)

File: client/src/pages/WeeklyTable.js [hunk 3]
```diff
@@ -74,12 +76,55 @@ const WeeklyTable = ({
...
-	console.log({ bookingsByRow });
+	const [currentMonday, setCurrentMonday] = useState(new Date(2022, 0, 17));
+	let week = [];
+	const year = currentMonday.getFullYear();
+	const month = currentMonday.getMonth();
+	let date = currentMonday.getDate();
 
+	for (let i = 0; i < 5; i++) {
+		// this will even work if we cross over into the next month!!
+		week.push(new Date(year, month, date + i).toISOString());
```
Comment (Benjamin-GitHub) [inline client/src/pages/WeeklyTable.js:87]: smart movement!

**Review by Benjamin-GitHub — APPROVED** [review]
Good job!

evrything looks fine to me.

File: client/src/pages/WeeklyTable.js (hunk 2 above)
Comment (rebecalsantos) [reply client/src/pages/WeeklyTable.js:101]: :)

File: client/src/pages/WeeklyTable.js (hunk 1 above)
Comment (rebecalsantos) [reply client/src/pages/WeeklyTable.js:83]: good question

**Pull request merged**